util/log/clog_file.go

```diff
@@ -0,0 +1,125 @@
+// Go support for leveled logs, analogous to https://code.google.com/p/google-clog/
+//
+// Copyright 2013 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
this file is pretty much just a copy. Might have moved a variable or two but nothing else.